iocage_lib/ioc_plugin.py

```diff
@@ -76,6 +77,26 @@ def __init__(self, release=None, jail=None, plugin=None, branch=None,
         self.keep_jail_on_failure = keep_jail_on_failure
         self.thickconfig = kwargs.pop('thickconfig', False)
         self.log = logging.getLogger('iocage')
+        self.git_repository = kwargs.get(
+            'git_repository'
+        ) or 'https://github.com/freenas/iocage-ix-plugins.git'
+
+        if not self.git_repository.endswith('.git'):
+            iocage_lib.ioc_common.logit(
+                {
+                    'level': 'EXCEPTION',
+                    'message': f'Git Repository URI is invalid.'
+                },
+                _callback=self.callback
+            )
+
+        self.git_destination = kwargs.get('git_destination')
+        if not self.git_destination:
+            self.git_destination = os.path.join(
+                self.iocroot, '.plugins', '_'.join(
+                    self.git_repository.split('/')[-2:]
```

Comment:
http://mygitrepo.com/somefolder/foo/bar could happen (not on github). rsplit('/', 1)[-1]  for safety?